include/zephyr/net/wifi_mgmt.h

```diff
@@ -316,6 +324,7 @@ void wifi_mgmt_raise_iface_status_event(struct net_if *iface,
 		struct wifi_iface_status *iface_status);
 void wifi_mgmt_raise_twt_event(struct net_if *iface,
 		struct wifi_twt_params *twt_params);
+void wifi_mgmt_raise_twt_state(struct net_if *iface, int twt_sleep_state);
```

Comment:
shouldn't the function name match the .c file ?

Reply:
Raised a PR for this https://github.com/zephyrproject-rtos/zephyr/pull/56825